src/setup_httpd.erl

```diff
@@ -29,16 +29,28 @@ handle_setup_req(#httpd{method='POST'}=Req) ->
     end;
 handle_setup_req(#httpd{method='GET'}=Req) ->
     ok = chttpd:verify_is_server_admin(Req),
-    case setup:is_cluster_enabled() of
+    Dbs = chttpd:qs_json_value(Req, "ensure_dbs_exist", setup:cluster_system_dbs()),
+    couch_log:notice("Dbs: ~p~n", [Dbs]),
+    case erlang:list_to_integer(config:get("cluster", "n", undefined)) of
+    1 ->
```

Comment:
Assumption here is that if n=1, you're on a single node cluster. This is probably valid.

If not we'll need to have `setup:enable_single_node` to set a new config value to key off of, e.g. `[cluster] single_node = true`.